sms_sponsorship/models/sms_child_request.py

```diff
@@ -34,8 +32,7 @@ class SmsChildRequest(models.Model):
     _rec_name = 'child_id'
     _order = 'date desc'
 
-    sender = Phone(partner_field='partner_id',
-                   country_field='country_id')
+    sender = fields.Char('Phone')
```

Comment:
The model should inherit from `phone.validation.mixin` and add a property `_phone_name_fields = ['sender']`